keyboards/modelh/chconf.h

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2020 QMK
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define CH_CFG_ST_TIMEDELTA                 0
+#define CH_CFG_USE_TM                       TRUE
+#define CH_CFG_USE_REGISTRY                 TRUE
+#define CH_CFG_USE_WAITEXIT                 TRUE
+#define CH_CFG_USE_CONDVARS                 TRUE
+#define CH_CFG_USE_CONDVARS_TIMEOUT         FALSE
+#define CH_CFG_USE_MESSAGES                 TRUE
+#define CH_CFG_USE_MAILBOXES                TRUE
+#define CH_CFG_USE_HEAP                     TRUE
+#define CH_CFG_USE_OBJ_CACHES               TRUE
+#define CH_CFG_USE_DELEGATES                TRUE
```

Comment:
Is there a particular reason for these?

Reply:
I'm not the original author. The original code was made for an [old version ](https://github.com/jhawthorn/qmk_firmware/commits/modelh_stm32)of QMK. I just ported the changes here.
About this file specifically, I included the parameters that were different from the default ones in `platforms/chibios/boards/common/configs/chconf.h`.

Reply:
Could you try removing this file and seeing if the board still works? I think most of these are pointless and inflate the file size for no reason; `CH_CFG_ST_TIMEDELTA` *may* be needed but usually isn't.